.github/workflows/automation-scripts/update-changelogs.mjs

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env zx
+
+"use strict";
+
+const fs = require('fs');
+const path = require('path');
+
+/****************************************************************
+* To run manually:
+* git checkout target branch (master or latest release); git pull
+* git checkout release/X.X.x; git pull
+* uncomment git checkout -b cmd and fix branch name
+* open PR into target branch
+*****************************************************************/
+
+const targetPath = "./temp-target-changelogs"
+const incomingPath = "./temp-incoming-changelogs"
+
+await $`mkdir ${targetPath}`
```

Comment:
A comment about zx might be helpful. People may be unfamiliar with the tool.

Reply:
done